src/main/java/com/hwr/hwrzeiterfassung/database/controller/HumanController.java

```diff
@@ -31,7 +34,25 @@
             return human.get().getProjects();
         }
         return (Iterable<Project>) new Project();
+    }
+
+    @GetMapping(path = "/name")
+    public @ResponseBody
+    Map<String, String> getHumanName(@RequestParam String email, @RequestParam String password) {
+
+        if (!loginController.validateLoginInformation(email, password)) {
+            throw new ResponseStatusException(HttpStatus.NOT_ACCEPTABLE, "User Credentials invalid");
+        }
+
+        var human = humanRepository.findById(email);
+        if (human.isEmpty()) {
+            throw new ResponseStatusException(HttpStatus.NOT_ACCEPTABLE, "Human don't exists");
+        }
 
+        Map<String, String> map = new HashMap<>();
+        map.put("firstName", human.get().getFirstName());
+        map.put("lastName", human.get().getLastName());
+        return map;
```

Comment:
Ist das nicht unnötig die Daten in eine Map zu legen. Return doch einfach direkt eine Human (FirstName, LastName)

Reply:
Nicht zwingend, denke ich, es sei denn, du brauchst im Frontend noch mehr als Vor- und Nachname. Falls nicht, ist eine Map meiner Meinung nach sinnvoll, weil wir nicht so viele unnötige Daten mitschicken, aber eine klare Zuordnung haben

Reply:
der Human im Backend hat noch mehre Daten als (FirstName, LastName) und deshalb schicke ich es als map